HomeWork_09/HW9.py

```diff
@@ -0,0 +1,12 @@
+
+text = int(input("Enter a digit: "))
+def my_generator():
+    f1, f2 = 0, 1
+    for i in range(text):
+        print(f1, end=" ")
+        f1, f2 = f2, f1 + f2
+    yield i
```

Comment:
Your generator should be able to give values one by one by calling `next` function or by putting it into for in construction. It your case, generator works like a function that looks for some element in the Fibonacci sequence. Moving `yield` inside of your `for in` block should fix this